rxmongo/src/main/scala/akka/contrib/persistence/mongodb/RxMongoReadJournaller.scala

```diff
@@ -147,7 +147,16 @@ class CurrentAllPersistenceIds(val driver: RxMongoDriver) extends IterateeActorP
   val temporaryCollection = driver.collection(temporaryCollectionName)
 
   override def cleanup() = {
-    driver.collection(temporaryCollectionName).drop().map(_ => ADone)
+    import reactivemongo.api.commands.bson.DefaultBSONCommandError
+    driver.collection(temporaryCollectionName).drop().recover {
+      // we ignore the "ns not found" error which is NOT filtered out by ReactiveMongo when trying to drop a non existing collection
+      // see https://github.com/ReactiveMongo/ReactiveMongo/issues/205
+      case commandError: DefaultBSONCommandError => {
+        commandError.errmsg match {
+          case Some(str) if str.equals("ns not found") => ()
```

Comment:
So this will cause a `MatchError` on miss, no?  What do you think about moving the filter out to the `PartialFunction` passed to `recover`?